src/client/Microsoft.Identity.Client/ManagedIdentity/EnvironmentVariables.cs

```diff
@@ -12,5 +12,6 @@ internal class EnvironmentVariables
         public static string PodIdentityEndpoint => Environment.GetEnvironmentVariable("AZURE_POD_IDENTITY_AUTHORITY_HOST");
         public static string ImdsEndpoint => Environment.GetEnvironmentVariable("IMDS_ENDPOINT");
         public static string MsiEndpoint => Environment.GetEnvironmentVariable("MSI_ENDPOINT");
+        public static string IdentityServerThumbprint => Environment.GetEnvironmentVariable("IDENTITY_SERVER_THUMBPRINT");
```

Comment:
What is the purpose of this validation [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/src/ServiceFabricManagedIdentitySource.cs#L96)

Reply:
The app service and service fabric have same environment variables. This one is used to identify the service fabric source.

Reply:
there seems to be some sort of cert validation. If we are not validating the cert and have a confirmation from Azure SDK on it then resolve this and merge

Reply:
Yes we discussed this in the last meeting we had. Basically they said it is upto us if we want to validate. The validation cannot be done in case a http client is passed by the user. Only when using a default http client we can validate the server certificate. So decided not to validate the server certificate.